core/src/main/java/org/apache/accumulo/core/clientImpl/AccumuloAccessUtils.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.clientImpl;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.accumulo.core.security.Authorizations;
+
+public class AccumuloAccessUtils {
+  public static org.apache.accumulo.access.Authorizations convert(Authorizations auths) {
+    List<String> authStrings = new ArrayList<>(auths.size());
+    for (byte[] authBytes : auths.getAuthorizations()) {
+      authStrings.add(new String(authBytes, UTF_8));
+    }
+
+    return org.apache.accumulo.access.Authorizations.of(authStrings);
```

Comment:
In all places where I have seen this used, it's in the form of:

```
AccessEvaluator.of(AccumuloAccessUtils.convert(Authorizations))
```

I'm curious if this would be better suited as a non-static method on `org.apache.accumulo.core.security.Authorizations`. Something like `toAccessAuthorizations`. At the very least it would make suggestions work in the IDE. Specifically, I'm thinking of:

```
AccessEvaluator.of(Authorizations.toAccessAuthorizations())
```

Reply:
I like that, its ok to have the accumulo access types in the accumulo API

Reply:
There's also an optimization that you could make if you do this, which is to store the Access Authorizations object after it's been computed. I don't know how many times `toAccessAuthorizations` would be called on a given Authorizations object, but if it's called more than once then it would save some time.

Reply:
even if they are not stored, computing them in a method on the object will be more efficient... currently it copies everything to a list and that step could be avoided

Reply:
> I like that, its ok to have the accumulo access types in the accumulo API

You might need to update core/pom.xml to allow the Authorizations object from accumulo-access. Two of the other types are already in that file as part of this PR.

Reply:
updated in 01322f2

Reply:
Changed the impl in d94937e because looking at the latest accumulo access code it no longer had a method I was calling that existed in the beta release.   Also opened apache/accumulo-access#70 based on these changes.